rpcs3/rpcs3qt/main_window.ui

```diff
@@ -641,6 +633,14 @@
     <string>Show Log/TTY</string>
    </property>
   </action>
+  <action name="showKernelExplorerAct">
```

Comment:
This is wrong. The "Show" menu is only for the main window, not for any child windows.

Reply:
Fixed.